cli/src/build.ts

```diff
@@ -329,15 +330,15 @@ export class BuildCommand extends Command {
       }
     }
 
-    if ((this.useZig || isCrossForLinux || isCrossForMacOS) && isZigExisted) {
+    if (useZig) {
```

Comment:
Note that `useZig = true` implies all of the previous conditional (it can only be set true if `(this.useZig || isCrossForLinux || isCrossForMacOS)` and zig existed)